src/reference/modules/reference/pages/jetty_server_configuration.adoc

```diff
@@ -82,265 +78,4 @@ you can just add corresponding lines at the end of `start.ini`:
 
 - `/var/rudder/tmp/jetty` is the directory where `Jetty` unzip web application `war` to run them.
 - `/var/rudder/run/rudder-jetty.pid`: file containing `Jetty` PID
-- `/opt/rudder/share/webapps` contains `RUDDER` `war` files.
-
```

Comment:
Why removing it? It was added because it was the only way for user to find where these things are in Rudder with our search tool, typically for lookup path "I need to change that param found in google, is it referenced in doc?"
cf the introduction commit: https://issues.rudder.io/issues/16677